reference/profiles.rst

```diff
@@ -217,6 +244,68 @@ The ``include()`` statement has to be at the top of the profile file:
     zlib:CC=/usr/bin/clang
     zlib:CXX=/usr/bin/clang++
 
+
+.. _profiles_configuration_data_operators:
+
+Configuration data operators
++++++++++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It's also possible to use some extra operators when you're composing different tools configurations:
+
+.. code-block:: text
+    :caption: global.conf
```

Comment:
This is not global.conf, but a profile, isn't it?
```suggestion
    :caption: profile_apple
```

Reply:
I wrote global.conf to show users that `global.conf` is the first configuration loaded (if it exists) even before the profiles. I can change it by another profile for sure but I found it interesting. Perhaps, it's not needed here...

Reply:
The thing is that this file contains [settings], [conf] sections, so it cannot be the ``global.conf`` file, it is confusing.

Reply:
you're right, then, if I delete those sections it could run this example, right? Or would it be confusing yet?

Reply:
Yes, if they are deleted, it would be a bit clearer.